### ISSUE TITLE
Explicitly document that `twisted.test.proto_helpers` is covered by the compatability policy.

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8857]
|[<img alt="tomprince's avatar" src="https://avatars.githubusercontent.com/u/283816?s=50" width="50" height="50">](https://github.com/tomprince)| @tomprince reported|
|-|-|
|Trac ID|trac#8857|
|Type|enhancement|
|Created|2016-10-10 07:33:17Z|
|Branch|https://github.com/twisted/twisted/tree/8857-proto_helpers-policy|

As discussed in [#6435](https://github.com/twisted/twisted/issues/6435), `twisted.test_proto_helpers` is effectively public, and should be moved out of a test package, and the API rethought with that in mind.

In the mean time, there are people actively using the code, so we should explicitly document that we will maintain compatibility for the things in it.

<details><summary>Searchable metadata</summary>

```
trac-id__8857 8857
type__enhancement enhancement
reporter__tomprince tomprince
priority__normal normal
milestone__None None
branch__8857_proto_helpers_policy 8857-proto_helpers-policy
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1476084797107326 1476084797107326
changetime__1476841242741200 1476841242741200
version__None None
owner__tom_prince tom.prince

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="tomprince's avatar" src="https://avatars.githubusercontent.com/u/283816?s=50" width="50" height="50">](https://github.com/tomprince)<a name="note_1"></a>|@tomprince commented|
|-|-|

https://github.com/twisted/twisted/pull/558

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_2"></a>|@adiroiban set owner to @tomprince|
|-|-|

Changes looks good.

Please update with trunk and merge.

Thanks!

**Comment by twisted-trac** [comment]
|<img alt="Tom Prince's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_3"></a>|Tom Prince set status to `closed`|
|-|-|

In changeset 2fca4b2ed05e52992e274f281dc614ff925cac8c
```
#!CommitTicketReference repository="" revision="2fca4b2ed05e52992e274f281dc614ff925cac8c"
Merge 8857-proto_helpers-policy: Document compatibility policy for proto_helpers explicitly.

Author: tomprince
Reviewer: adiroiban
Fixes: #8857
```
